gradle.properties

```diff
@@ -0,0 +1,8 @@
+version=2.1.1-SNAPSHOT
+# Knot.x Docker image version (https://hub.docker.com/r/knotx/knotx)
+knotx.version=2.1.0
```

Comment:
I think that should alway be the same as `version`.

Reply:
Done